e2e/steps_file_LRspec.js

```diff
@@ -15,8 +15,6 @@ const addAnotherDefendant = require('./pages/createClaim/addAnotherDefendant.pag
 const respondent2SameLegalRepresentative = require('./pages/createClaim/respondent2SameLegalRepresentative.page');
```

Comment:
This file is for specified end to end tests only. There should be another one for unspecified tests called steps_file.js.

In the api folder, there are also a couple steps files for the api tests, you might have to make some changes there.

Reply:
Done